frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -408,6 +408,17 @@ const plugin: Plugin<ConsumedExtensions> = [
       disallowed: [ODF_MANAGED_FLAG, ODF_MODEL_FLAG],
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      path: `/ocs-dashboards`,
+      loader: () => import('./components/dashboards/ocs-system-dashboard').then((m) => m.default),
+    },
+    flags: {
+      required: [MCG_FLAG],
+      disallowed: [OCS_FLAG, ODF_MODEL_FLAG],
+    },
+  },
```

Comment:
I don't remember why I added this. Maybe something was in my mind for 4.10 PR, but, this will not affect this BZ. Should I remove this ?

Reply:
This looks something that would be required for OCS 4.8 support. AFAIK this is required to support OCS 4.8 standalone MCG.  MCG standalone in 4.8 doesn't create StorageCluster.

Reply:
So, it is good to have this fix in this backport PR ? (OCP 4.9 + OCS 4.8 is valid combination)
We can remove this from 4.10